manageability-docker/README.md

```diff
@@ -13,11 +13,20 @@ This folder contains the build instructions for creating mangeability docker con
 ## HOW TO BUILD INB IMAGE AND START THE CONTAINER
 
 * Unzip the package(inb_azure_container.zip/inb_tb_container.zip).
-* If planning to use the sample_customer_mqtt_client provided, open the cloud_source file and uncomment the lines specified in the file.
+* If planning to use the `sample_customer_mqtt_client` provided please refer the section below. 
 * If using Thingsboard, edit the tb_conf_file and fill in the respective Thingsboard server details and device tokens along with tls certificate of thingsboard server if TLS is enabled on the Thingsboard server. If using TLS for Thingsboard uncomment the line 'cp /src/thingsboard.pub.pem /etc/intel-manageability/secret/cloudadapter-agent/thingsboard.pub.pem' in the cloud_source file.
 * If using Azure, edit the azure_conf_file and fill in the Primary SAS Key, device name, scope id.
 * Now run ./run.sh which builds the docker image and starts the container.
 
 
 If an error such as 'unable to resolve' or a DNS error or 'unable to look up' is seen near the start of the build, follow the instructions under https://docs.docker.com/install/linux/linux-postinstall/ --> "DISABLE DNSMASQ".  This can occur in some Linux distributions that put 127.0.0.1 in /etc/resolv.conf.
 
+
+## HOW TO USE THE SAMPLE CUSTOMER MQTT CLIENT
+* Unzip the package built (inb_azure_container.zip/inb_tb_container.zip)
+* Open the cloud_source file and uncomment the lines specified in the file. 
+* Create a directory for storing certs on host device and edit the mqtt_client.py file variables DEFAULT_MQTT_CERTS, CLIENT_CERTS, CLIENT_KEYS to point to your local directory with the same cert,key names as seen in the file Ex: /home/harsha/certs/cmd-program.crt to be modified to <your_directory>/cmd-program.crt.
+* Edit the run.sh file’s docker run command ‘-v /home/harsha/certs:/var/certs’ to point it to the directory like ‘-v <your_directory>:/var/certs’
+* Edit the tb_conf_file with your configuration and run ./run.sh with sudo.
```

Comment:
Add the word 'file' after tb_conf_file to complete the sentence.  Maybe put the ./run.sh on the next line and put it in the ``` marks so it can be easily copied.

Here's how I'd put the 2 lines.
* Edit the tb_conf_file file with your configuration.
*  Run ```sudo ./run.sh```